applications/ingress-nginx/values-usdfdev.yaml

```diff
@@ -0,0 +1,21 @@
+ingress-nginx:
+  controller:
+    config:
+      compute-full-forwarded-for: "true"
+      large-client-header-buffers: "4 64k"
+      proxy-body-size: "100m"
+      proxy-buffer-size: "64k"
+      ssl-redirect: "true"
+      use-forwarded-headers: "true"
+    service:
+      externalTrafficPolicy: Local
+      loadBalancerIP: "134.79.138.113"
+    podLabels:
+      hub.jupyter.org/network-access-proxy-http: "true"
+
+vault_certificate:
```

Comment:
This should now be `vaultCertificate` (although you're setting it to the default so you could just delete it).